subprojects/internal-integ-testing/src/main/groovy/org/gradle/integtests/fixtures/jvm/JavaToolchainBuildOperationsHelper.groovy

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.integtests.fixtures.jvm
+
+import org.gradle.api.internal.tasks.execution.ExecuteTaskBuildOperationType
+import org.gradle.api.internal.tasks.execution.ResolveTaskMutationsBuildOperationType
+import org.gradle.integtests.fixtures.BuildOperationsFixture
+import org.gradle.internal.jvm.inspection.JvmInstallationMetadata
+import org.gradle.internal.operations.BuildOperationType
+import org.gradle.internal.operations.trace.BuildOperationRecord
+import org.gradle.jvm.toolchain.internal.operations.JavaToolchainUsageProgressDetails
+
+class JavaToolchainBuildOperationsHelper {
```

Comment:
What we often do is we make those classes a Groovy `trait`, call them `*Fixture` and them mix them into the test class via `implements`. You can take a look at for example `DirectoryBuildCacheFixture`.

Reply:
Interesting, I will give this a try

Reply:
Done